docs/definitions/class.md

```diff
@@ -0,0 +1,19 @@
+# Class
+
+In ML, a **class** (or category) is a possible value a discrete **output**
+variable can take. It is therefore used in the context of classification (or
+categorization) problems.
+
+Classes allow to differentiate (or to classify)
+the data – or a subset of the data – into a **finite** set of options.
+The different possible options an output variable can take are often
+referred to as classes.
+
+**Note**
+Classes and labels, although sometimes used indistinctively, do not share the
+exact same definition. Be sure to check the
+[`ml:label` definition](/definitions/label). A class is the possible value a
```

Comment:
These links do not seem to work in Github in the current branch (maybe they're not meant to?)

See https://github.com/mlcommons/croissant/blob/pierrot0/labels/docs/definitions/class.md
Clicking on `ml:label definition` leads me to the non-existing 
https://github.com/mlcommons/croissant/blob/pierrot0/labels/definitions/label 
Instead of 
https://github.com/mlcommons/croissant/blob/pierrot0/labels/docs/definitions/label.md

Reply:
`](/definitions/label)` -> `]((/docs/definitions/label))`